src/gui/fspathedit.cpp

```diff
@@ -154,7 +154,9 @@ void FileSystemPathEdit::FileSystemPathEditPrivate::modeChanged()
     switch (m_mode) {
     case FileSystemPathEdit::Mode::FileOpen:
     case FileSystemPathEdit::Mode::FileSave:
-        pixmap = QStyle::SP_DialogOpenButton;
```

Comment:
Did you intentionally remove this line? Isn't it needed for other platforms (except the Windows)?

Reply:
Yes, `pixmap` already have a default above: `QStyle::StandardPixmap pixmap = QStyle::SP_DialogOpenButton;`